src/main/webapp/dev/js/common/feedbackResponseComments.js

```diff
@@ -169,7 +169,7 @@ function showResponseCommentAddForm(recipientIndex, giverIndex, qnIndex, section
     }
     $(`#showResponseCommentAddForm${id}`).show();
 
-    $(`#responseCommentAddForm${id}`).empty();
+    $(`#responseCommentAddForm${id}`).html('<br data-mc-bogus="1">');
```

Comment:
Maybe something like this: ``tinymce.get(`#responseCommentAddForm${id}`).setContent('');``?